File: 2997-minimum-number-of-operations-to-make-array-xor-equal-to-k/2997-minimum-number-of-operations-to-make-array-xor-equal-to-k.java
class Solution {
    public int minOperations(int[] nums, int k) {
        int ans = nums[0];
        int n = nums.length;
        for (int i = 1; i < n; i++) {
            ans ^= nums[i];
        }
        int count = ans ^ k;
        return Integer.bitCount(count);
    }
}

File: LeetCode/2997-minimum-number-of-operations-to-make-array-xor-equal-to-k/2997-minimum-number-of-operations-to-make-array-xor-equal-to-k.cpp
class Solution {
public:
    int minOperations(vector<int>& nums, int k) {
        int ans = nums[0];
        int n = nums.size();
        for (int i = 1; i < n; i++) {
            ans = ans ^ nums[i];
        }
        int count = (ans ^ k);
        return __builtin_popcount(count);
    }
};